FfaFeaturePkg/Applications/FfaPartitionTest/FfaPartitionTestApp.c

```diff
@@ -1305,6 +1311,73 @@ FfaMiscTestTpmReopenLocality (
   return UNIT_TEST_PASSED;
 }
 
+/**
+  This routine tests the FFA_NS_RES_INFO_GET command.
+**/
+UNIT_TEST_STATUS
+EFIAPI
+FfaMiscTestFfaNsResInfoGet (
+  IN UNIT_TEST_CONTEXT  Context
+  )
+{
+  EFI_STATUS                     Status;
+  FFA_TEST_CONTEXT               *FfaTestContext;
+  UINT32                         WrittenSize;
+  UINT32                         RemainingSize;
+  VOID                           *TxBuffer;
+  VOID                           *RxBuffer;
+  UINT64                         TxSize;
+  UINT64                         RxSize;
+  FFA_RESOURCE_INFO_DESC_HEADER  *ResourceDescHeader;
+  FFA_ADDRESS_MAP_DESC           *AddressMapDescArray;
+  UINT32                         Index;
+
+  DEBUG ((DEBUG_INFO, "%a: enter...\n", __func__));
+
+  FfaTestContext = (FFA_TEST_CONTEXT *)Context;
+  UT_ASSERT_NOT_NULL (FfaTestContext);
+
+  // Invoke FFA_NS_RES_INFO_GET
+  Status = FfaNsResInfoGet (0, FFA_NS_RES_INFO_GET_REQ_START_FLAGS, &WrittenSize, &RemainingSize);

```

Comment:
FYI, once this is picked up from the mainline, it will try to run this test during CI, which I assume will fail?

I would recommend add this test, but in the meantime add something as the test prerequisite, so that it will skip the test if the hafnium does not support it.